awscrt/io.py

```diff
@@ -50,28 +64,49 @@ def __init__(self):
 
         self.min_tls_ver = TlsVersion.DEFAULT
 
-    def override_default_trust_store(self, ca_path, ca_file):
+    @classmethod
```

Comment:
This doesn't belong here I don't think?

Reply:
Why not?

Reply:
Because it makes the method static, but the method is written like an instance method. If this is intended to be static, you should rename `self` to `cls`.

Reply:
@staticmethod does that, this is @classmethod

Reply:
Discussed offline, fixed.